### PR TITLE
General code simplifications

### PR DESCRIPTION
**Pull request by ferhatelmas**
* unnecessary bool constant and error check

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -481,10 +481,8 @@ func GetDeltaConfig() (max_deltas int, from_full bool) {
 	if hasOrigin {
 		switch origin {
 		case "LATEST":
-			break
```
Comment (x4m) [inline commands.go]: Are you sure it's OK to remove this?

**Comment by x4m** [comment]
Thanks! Most of refactorings look great!
I just have a question regarding switch\break, is it really ok to do so?

File: commands.go (hunk 1 above)
Comment (ferhatelmas) [reply commands.go]: Yes, it's fine. https://play.golang.org/p/e4thhtWzYcw

Comment (x4m) [reply commands.go]: Cool

**Comment by x4m** [comment]
Thanks, pushed

**Pull request merged**